tests/suites/authorized_keys/bootstrap.sh

```diff
@@ -28,10 +28,11 @@ run_bootstrap_authorized_keys_loaded() {
 	ssh-keygen -t ed25519 -f "$extra_key_file" -C "isgreat@juju.is" -P ""
 
 	HOME="${SUB_TEST_DIR}"
+	JUJU_DATA="${juju_home_dir}"
```

Comment:
Are you sure this works? This variable is not actually being exported here, just set locally. I don't think the juju cli would be aware of it.

Reply:
Hence the shell check error:
```
     | In tests/suites/authorized_keys/bootstrap.sh line 77:
    | JUJU_DATA="${juju_home_dir}"
    | ^-------^ SC2034 (warning): JUJU_DATA appears unused. Verify use (or export if used externally).
```

Reply:
It most definitely does work. The variable set to the scope of this func. Exporting it would affect the rest of the env which we do not want to do.

Reply:
Fixed up.